### ISSUE TITLE
Selecting expiration date seems to broken when creating products on iOS

### ISSUE THREAD

**Issue by taalas** [PantryParty/pantry_party#116]
Whenever I scan a new product and click on the expiration date field, the field disappears but not date chooser is displayed. There seems to be a scrollbar displayed but this too is without function.

**Comment by arjes** [comment]
This is in the form right? Not the guided mode (when you click the yellow
bar on the top)?

**Comment by taalas** [comment]
Yes, exactly. The form after clicking an entry. I do have some other weird UI issues, but this seems the only one that is breaking the UI for me, I think.

**Comment by arjes** [comment]
@taalas & @rda1990 I'm trying to reproduce the error but it isn't , can you tell me what version of iOS you are running?

**Comment by rda1990** [comment]
> @taalas & @rda1990 I'm trying to reproduce the error but it isn't , can you tell me what version of iOS you are running?

I am using iOS 14.6 on an iPhone 12 Pro Max

**Comment by arjes** [comment]
@rda1990 thanks I'll see if I can reproduce it in a simulator. 

In the short term, clicking the yellow bar on screen will walk you through a guided product creation. It isn't as detailed as the form you are having problems on but it should work.

**Comment by arjes** [comment]
I'm able to reproduce it, this looks like an error in one of the upstream libraries so it may take a few to track down.

**Comment by rda1990** [comment]
okay, but I can't add items from the app because it does not let me change the date any where.

**Comment by arjes** [comment]
@rda1990 check on this workflow: https://pantryparty.app/docs/basic_usage/product-creation.html Step 6 (https://pantryparty.app/docs/basic_usage/product-creation.html#step-6-shelf-life-before-open) will allow you to set the expiration date as "Days from now".

You access this workflow by tapping this banner: https://pantryparty.app/docs/basic_usage/purchase.html#product-creation

**Comment by arjes** [comment]
@rda1990 & @taalas. Thank you for your bug report. I have a fix ready and I'll be working to get int published in the next couple day or so (Sometimes apple can be slow).

**Comment by rda1990** [comment]
Thanks for the fix and your time
